docker/run_tests.sh

```diff
@@ -1,47 +1,55 @@
 #!/bin/bash
 
-# $1: compiler (gcc-4.8, gcc-4.9, gcc-5, gcc-6, clang-3.8, clang-3.9, clang-4.0)
+# $1: compiler (gcc-4.8, gcc-5, gcc-6, gcc-7, clang-4.0, clang-5.0)
 # $2: moab version (5.0, master)
-# $3: geant4 version (10.01.p03, 10.02.p03, 10.03.p01)
+# $3: build Dag-Geant4 (OFF, ON)
 
 set -e
 
 source /root/etc/$1.env
-hdf5_version=1.8.13
 moab_version=$2
-geant4_version=$3
+build_daggeant4=$3
+hdf5_version=1.8.13
+geant4_version=10.04
 
 DAGMC_dir=${install_dir}/DAGMC-moab-${moab_version}
 
 export PATH=${install_dir}/hdf5-${hdf5_version}/bin:${PATH}
 export PATH=${install_dir}/moab-${moab_version}/bin:${PATH}
-export PATH=${install_dir}/geant4-${geant4_version}/bin:${PATH}
 export PATH=${DAGMC_dir}/bin:${PATH}
 export LD_LIBRARY_PATH=${install_dir}/hdf5-${hdf5_version}/lib
 export LD_LIBRARY_PATH=${install_dir}/moab-${moab_version}/lib:${LD_LIBRARY_PATH}
-export LD_LIBRARY_PATH=${install_dir}/geant4-${geant4_version}/lib:${LD_LIBRARY_PATH}
 export LD_LIBRARY_PATH=${DAGMC_dir}/lib:${LD_LIBRARY_PATH}
 
 # Run astyle to see if there are any differences
-${DAGMC_dir}/tools/astyle --options=${DAGMC_dir}/tools/google.ini \
-                          --exclude=gtest \
-                          --exclude=tools/astyle \
-                          --exclude=mcnp/mcnp5/Source \
-                          --exclude=mcnp/mcnp6/Source \
-                          --ignore-exclude-errors \
-                          --recursive \
-                          --verbose \
-                          --formatted \
-                          "*.cc" "*.cpp" "*.h" "*.hh" "*.hpp"
-git diff --exit-code  # Exit if astyle found diffs
+astyle_deb=astyle_3.0.1-1ubuntu1_amd64.deb
```

Comment:
So after this PR, we still build `astyle` with DAGMC but don't use it to test right?

Reply:
That's fine by me btw. The removal of `astyle` is its own PR.

Reply:
Yes. Also you'll see that we have to download a .deb straight from the internet instead of using the package manager. This is because we need to use astyle version 3, while all ubuntu versions before 18.04 can only get version 2.06.

Reply:
Personally, I don't think that's a big deal but why do we need version 3?

Reply:
Version 2 was segfaulting somewhere in our code